util/test/test_blocktime.py

```diff
@@ -147,14 +153,63 @@ def test_getstfinBlocks():
     assert fin == 0
 
 
+@enforce_types
+def test_ethTimestamptoBlock():
+    networkutil.connect(1)
+
+    _chain = brownie.network.chain
+    ts = _chain[-5000].timestamp
+    block = _chain[-5000].number
+
+    guess = ethTimestamptoBlock(_chain, ts)
+
+    assert guess == approx(block, 10)
+
+    networkutil.connect(networkutil.DEV_CHAINID)
+
+
+@enforce_types
+def test_ethFindFirstThuBlock():
```

Comment:
For this test, I have the same questions about network setup and teardown as in the previous test.